### ISSUE TITLE
Set task priority

### ISSUE THREAD

**Issue by Valdnet** [nextcloud/tasks#833]
### Steps to reproduce
1. Add a new task.
2. Set priority.
3. The slider should work the other way around.

### Expected behaviour
The slider should work from low priority to largest. This is more understandable to the user. The more that it starts with no priority and the number that indicates should be relevant to the priority.

### Actual behaviour
The slider works the other way around.

![obraz](https://user-images.githubusercontent.com/47037905/73138957-b5420280-4068-11ea-8542-d7097b1a02e5.png)

![obraz](https://user-images.githubusercontent.com/47037905/73138930-554b5c00-4068-11ea-9886-010a42f69a48.png)

**Operating system**:
Win7

**Nextcloud version:** (see Nextcloud admin page)
18.0.0

**Tasks version:** (see Nextcloud apps page)
0.11.3

**Comment by raimund-schluessler** [comment]
Well, RFC5545 specifies a value of "9" as low priority, "1" as the highest priority and "0" as "undefined"/no priority: https://tools.ietf.org/html/rfc5545#section-3.8.1.9, so this is the intended behaviour. The order would then be `0, 9, 8, ..., 2, 1` from low to high priority. There is not much we can do about that, I think.

**Comment by amaddeus** [comment]
Have you given up on changing this?

**Comment by raimund-schluessler** [comment]
As said above, this is the intended behavior by RFC 5545.

**Comment by amaddeus** [comment]
Because this is the intended behavior by RFC 5545, is this impossible to change/ not worth changing?

**Comment by raimund-schluessler** [comment]
We could of course map the the RFC priority values to strictly monotonous values with e.g. 9 being the highest priority and 0 as the lowest / no priority. And indeed we did that in previous versions. But then we got complaints that this is not as the RFC intends it to be. So either way we do it, some people will not like it. That is why I decided to follow the RFC here and use the values as they are specified.
